processes/development/example-mapping-and-estimating-process.md

```diff
@@ -0,0 +1,215 @@
+
+     Do not wait 25 minutes to call it. As soon as the above behaviour is noted, move on
+
+  ##### How to handle questions/spikes
+
+  - Capture them on a card (on StoriesOnBoard)
+  - Assign someone responsible for finding answers
+  - (at the end of each session) Export cards to Trello (using the StoriesOnBoard sync feature)
+  - Research and address questions off-line, on card
+
+  ##### How to handle Client participation/involvement in the "BDD" aspect
+
+  - `Task`-level cards that are red-flagged and require Client participation in the Kickoff Process should be earmarked for meetings with client. 
+  - Use the standard Example Mapping process with the client involved to reveal rules and examples, resolve initial questions and reveal new ones
+  - Identify all questions including those that are the *Client's responsibility* to research and answer. Create question cards as normal
+
+- #### Create `Development` Cards
+
+  ##### For each `Task`-level card and completed Example Map sheet
+
+  - Break the task into 1 or more `Development`-level cards
+  - `Development`-level cards are:
+    - Smaller but complete chunks of functionality
+    - self-contained, *demonstrable*, ***testable***
+    - accompanied one or more rules and examples
+  - `Development`-level cards will be
+    - Estimated using story points
+    - Imported from StoriesOnBoard into Trello as work cards
+  - Each *Rule* naturally represents a potential way to split the task (ie. success rules vs failure rules)
+  - Look for natural seams and groupings in rules and the examples used to represent them, but don't get fixated on `1 rule per Dev card`
+  - 1-pager [How to Split a Story](https://drive.google.com/a/okgrow.com/file/d/1azKBvlOEnkA65t__vu_qXXY7b05WgX5b/view?usp=sharing)
+  - Good examples
+    - Success scenario/happy path VS failure scenario/error paths
+    - Simple (most common scenario) VS alternate methods (pay in cash vs credit)
+    - Handle potential count preconditions  (a list with 1 vs many vs none) if interesting or complex
+  - Bad card examples
+    - "Schema Design"
+    - "Database Design"
+    - "Menu Item UI"
+    - "Menu Item Query"
+
+- #### Add Estimates to `Development`-level Cards
+
+  ##### Create a POINTS BASELINE as a group
+
+  - Look through your collection of `dev`cards and agree on an exemplar of a 3-point card
+  - Do the same and find 5-point card
+  - All other cards are measured *relative* to these 2 cards
+
+  ##### Notes about thinking about cards
+
+  - Each card represents the *full scope of work*
+    - GraphQL / API
+    - UI implementation / design tweaks
+    - Routing
+    - Database
+    - Testing
+    - Done Done
+  - *The above does not mean everything gets bumped to next tier*
+  - It *does* mean that 1's are *very small* and very likely making changes / additions to existing features
+  - ***everything* is relative**, you should be considering *all* the work for *every* card
```

Comment:
I'd add a note: 

`Don't give higher point values on small tasks to be "conservative." That will distort the estimate, giving those tasks too much time relative to other work that needs to be done. Focus only on relative size`

Reply:
Nice!  Adding!